app/src/main/java/com/mathewsachin/fategrandautomata/root/SuperUser.kt

```diff
@@ -47,7 +50,7 @@ class SuperUser : AutoCloseable {
     /**
      * Writes a line to the shell.
      */
-    private fun writeLine(Line: String) {
+    fun writeLine(Line: String) {
```

Comment:
`sendCommand` and `waitForCommand` aren't being used anymore.
should we delete them?

Reply:
Those functions and a common `SuperUser` class were created because I tried out Gestures using root instead of AccessibilityServices ([RootGestures.kt](https://github.com/Fate-Grand-Automata/FGA/blob/b618fe1e042408dda3a3933d1262a6da208818e5/app/src/main/java/com/mathewsachin/fategrandautomata/root/RootGestures.kt)), which would've allowed the app to run on Android 5 and 6. But, the performance wasn't good, so I left it out since every emulator supports Android 7 anyways, and the app can't be backported to Android 5 or 6 due to a lot of APIs we use now.

We don't need those two functions now, so I guess it's fine to remove.

Reply:
I removed `sendCommand`, but kept `waitForCommand` because it contains a piece of knowledge that could be useful to future readers 🤔